### PR TITLE
Add Logviewer

### PR DESCRIPTION
**Pull request by Williangalvani**
image is up at williangalvani/companion-core:logviewer

it is served by nginx at /logviewer, and isn't picked up by the helper as it isn't an independent server.

### CODE REVIEW

File: core/Dockerfile [hunk 1]
```diff
@@ -22,6 +22,7 @@ RUN /home/pi/tools/dnsmasq/bootstrap.sh
 RUN /home/pi/tools/filebrowser/bootstrap.sh
 RUN /home/pi/tools/linux2rest/bootstrap.sh
 RUN /home/pi/tools/nginx/bootstrap.sh
+RUN /home/pi/tools/logviewer/bootstrap.sh
```
Comment (patrickelectric) [inline core/Dockerfile:25]: The list is in alphabetical order, can you fix logviewer position ?

File: core/tools/nginx/nginx.conf [hunk 2]
```diff
@@ -113,6 +113,10 @@ http {
             proxy_pass http://127.0.0.1:81;
         }
 
+        location /logviewer {
+            root /home/pi/tools;
```
Comment (patrickelectric) [inline core/tools/nginx/nginx.conf:117]: Should it be /home/pi/tools/logviewer ?

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: core/Dockerfile (hunk 1 above)
Comment (Williangalvani) [reply core/Dockerfile:24]: done

File: core/tools/nginx/nginx.conf (hunk 2 above)
Comment (Williangalvani) [reply core/tools/nginx/nginx.conf:117]: no, the current way the full path ends up at /home/pi/tools/logviewer (root + location)

Comment (patrickelectric) [reply core/tools/nginx/nginx.conf:117]: Thanks! That was what I was imagining

**Comment by patrickelectric** [comment]
Just a last thing, the commits message should have a `core:` prefix.
image

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**